OpenMage/magento-lts
# Admin theme issue (overlapping menu items)

### Description (*)

Small CSS update for the adminhtml. Removed the `z-index` from the `active` class used on the top menu bar to prevent the active item from rendering on top of the hover menu in some instances.

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes OpenMage/magento-lts#3394

### Manual testing scenarios (*)

1. Have enough top-level menu items to wrapped menu to more than 1 line
2. Open a page from a menu on the bottom row of menu items
3. Notice that the selected menu is now highlighted
4. Hover over a menu item ABOVE the highlighted menu
5. The pop-up/hover menu appears on top of the highlighted active menu

### Questions or comments

This is my first time working with `sass`.
I don't understand why it has changed various colours from hex to hsl representation, and added an additional decimal point to rgb values also.
How do I correct this? Do we care about this?

### Contribution checklist (*)
